time/time.go

```diff
@@ -3,7 +3,10 @@ package time
 import "time"
 
 // DeisDatetimeFormat is the standard date/time representation used in Deis.
-const DeisDatetimeFormat string = "2006-01-02T15:04:05MST"
+const DeisDatetimeFormat  string = "2006-01-02T15:04:05MST"
+// Different format to deal with the pyopenssl formatting
+// http://www.pyopenssl.org/en/stable/api/crypto.html#OpenSSL.crypto.X509.get_notAfter
+const DeisDatetimeFormat2 string = "2006-01-02T15:04:05"
```

Comment:
nit: can you name this something that indicates it's a pyopenssl format. maybe `PyOpenSSLTimeDateTimeFormat`?